fastlane/Fastfile

```diff
@@ -47,11 +47,12 @@ lane(:execute_tests) do
   version = local_version # has to be outside of the `Dir.chdir`
 
   # Verifying the --help command
-  #
-  Dir.chdir("..") do
-    content = sh("PAGER=cat bin/fastlane --help")
-    ["--version", "https://fastlane.tools", "fastlane"].each do |current|
-      UI.user_error!("--help missing information: '#{current}'") unless content.include?(current)
+  unless Helper.windows?
+    Dir.chdir("..") do
```

Comment:
I believe PAGER is set for commander.

Here: https://github.com/commander-rb/commander/blob/927d6418fbb027fc8e0dcfd6db48deea73f15168/lib/commander/user_interaction.rb#L274-L300

Have you tried having a different command on windows, e.g. just `bin/fastlane --help`?

Given that Windows doesn't have tty, paging should not be needed.

Reply:
I could make it do something with `cd bin && fastlane --help`, `bin/fastlane` it doesn't like. Let's try this.

Reply:
Yes, that works! Thanks for the idea. Is the code I used to differentiate the commands ok?